docs/ref/amend.md

```diff
@@ -73,11 +73,12 @@ q).[1 2; (); :; 3 4 5]
 3 4 5
 q).[1 2; (); ,; 3 4 5]
 1 2 3 4 5
-q)@[1 2; ::; ,; 3 4 5]
-1 2 3 4 5
 
 q)@[1 2; ::; *; 3 4]
 3 8
+@[(1 2;4 5); ::; ,; 3 6]
```

Comment:
Just needs a 'q)' appended to show which lines typed into console e.g.
```q)@[(1 2;4 5); ::; ,; 3 6]```

Thanks